File: dotnet/src/Model/Enemy/Units/ESoldier.cs
using System;

namespace PlanetCS1 {
    class ESoldier : Soldier {
        public ESoldier(Location location) {
            this.loc = location;
            this.use = true;
            this.drive = true;
            this.owner = 2;
            this.icon = "eS";
        }
    }
}

File: typescript/src/PlanetTS1/Model/Enemy/Units/ESoldier.ts
import { ALocation as ALocation } from "../../Utilities"

export class ESoldier extends Soldier {
    loc : ALocation
    use : boolean
    drive : boolean
    owner : number
    icon : string
    constructor(location : ALocation) {
        super()
        this.loc = location
        this.use = true
        this.drive = true
        this.owner = 2
        this.icon = "eS"
    }
}